socorro/processor/signature_utilities.py

```diff
@@ -970,3 +970,31 @@ def _action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
             "shutdownhang | %s" % processed_crash['signature']
         )
         return result
+
+
+#==============================================================================
+class SignatureJitCategory(Rule):
+    """replaces the signature if there is a JIT classification in the crash"""
+
+    #--------------------------------------------------------------------------
+    def version(self):
+        return '1.0'
+
+    #--------------------------------------------------------------------------
+    def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
+        return (
+            processed_crash.get('classifications') and
+            processed_crash.classifications.get('jit') and
+            processed_crash['classifications']['jit'].get('category')
```

Comment:
you could have done `processed_crash.get('classifications', {}).get('jit', {}).get('category')` :)
or 
```python
try:
    return processed_crash.classifications.jit.category
except KeyError:
    return 
```